tests/lib/rules/getter-return.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * @fileoverview Enforces that a return statement is present in property getters.
+ * @author Aladdin-ADD(hh_2013@foxmail.com)
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/getter-return");
+const RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+
+// data is not working, so specify a name: "getter 'bar'"
+const name = "getter 'bar'";
+const noReturnMessage = `Expected to return a value in ${name}.`;
+const noLastReturnMessage = `Expected to return a value at the end of ${name}.`;
+const parserOptions = { ecmaVersion: 6 };
+const options = [{ allowImplicit: true }];
+
+ruleTester.run("getter-return", rule, {
+
+    valid: [
+
+        // test obj: get, option: {allowImplicit: false}
+        { code: "var foo = { get bar(){return true;} };" },
+        { code: "var foo = { get bar(){return;} };" },
+        { code: "var foo = { bar: function(){return true;} };" },
+        { code: "var foo = { bar: function(){return;} };" },
+        { code: "var foo = { bar(){return true;} };", parserOptions },
+        { code: "var foo = { bar(){return;} };", parserOptions },
+        { code: "var foo = { bar(){~function (){}();return;} };", parserOptions },
```

Comment:
I find a mistake, the default option is allowImplicit : false, so it means we cannot return a value implicitly. in this case, should cause an error. thoughts? @kaicataldo @ljharb